client/components/ChartBox.tsx

```diff
@@ -346,7 +346,7 @@ const ChartBox: React.FC<IProps> = ({ queries, setDaysFn }) => {
                     Depth
                 </button>
                 <button onClick={() => chartThreeFn()} className="chartBtn" type="button">
-                    Complexity
```

Comment:
Why was this shortened? I think it should say `Complexity`

Reply:
It is a very long word. It was either to shorten it or break it for responsive desing

Reply:
Can we just shrink the font as part of the responsive design so that the whole word is displayed